website/gatsby-node.js

```diff
@@ -1,12 +1,30 @@
-/**
- * TODO: For some reason this doesn't work, get it to work :-(
- */
 exports.createPages = ({graphql, actions}) => {
   const {createRedirect} = actions;
-  createRedirect({
-    fromPath: '/all-options/',
-    toPath: '/all-props/',
-    isPermanent: true,
-    redirectInBrowser: true,
+
+  const paths = [
+    'creating-tooltips',
+    'customizing-tooltips',
+    'all-props',
+    'html-content',
+    'themes',
+    'animations',
+    'tippy-instance',
+    'methods',
+    'lifecycle-hooks',
+    'ajax',
+    'accessibility',
+    'addons',
+    'plugins',
+    'misc',
+    'faq',
+    'motivation',
+  ];
+
+  paths.forEach(path => {
+    createRedirect({
+      fromPath: `/${path}/`,
+      toPath: `/v5/${path}/`,
+      isPermanent: true,
```

Comment:
Are you sure you want to lock those paths to v5 docs? This way we would have to keep those pages basically forever :P

Reply:
The v5 warning messages link to these so unfortunately yes. Every major will now have a `/vN/` prefix for the docs.